pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "prometheus_client"
+version = "0.22.0"
+description = "Python client for the Prometheus monitoring system."
+readme = "README.md"
+license = { file = "LICENSE" }
+requires-python = ">=3.9"
+authors = [
+    { name = "The Prometheus Authors", email = "prometheus-developers@googlegroups.com" },
```

Comment:
This should be the only substantive change, moving from a specific author to the Promtheus developers as a whole.